src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder stb = new StringBuilder();
```

Comment:
stb - is't very informative naming, let's change it to something more meaningful

Reply:
done